src/mcts/params.cc

```diff
@@ -269,6 +275,37 @@ const OptionId SearchParams::kDrawScoreWhiteId{
 const OptionId SearchParams::kDrawScoreBlackId{
     "draw-score-black", "DrawScoreBlack",
     "Adjustment, added to a draw score of a black player."};
+const OptionId SearchParams::kWDLRescaleRatioId{
+    "wdl-rescale-ratio", "WDLRescaleRatio",
+    "Rescales the logistic WDL scale by the given ratio."};
+const OptionId SearchParams::kWDLRescaleDiffId{
+    "wdl-rescale-diff", "WDLRescaleDiff",
+    "Shifts the logistic WDL mean by diff in white's favor."};
+const OptionId SearchParams::kWDLContemptId{
+    "wdl-contempt", "WDLContempt",
+    "The simulated rating advantage for the WDL conversion."};
+const OptionId SearchParams::kWDLContemptAttenuationId{
+    "wdl-contempt-attenuation", "WDLContemptAttenuation",
+    "This scales the given Elo advantage used for contempt."};
```

Comment:
Basic idea: real Elo difference is given through `WDLContempt`; the attenuation factor is applied to turn this into a "simulated Elo difference", as it might make sense to "underestimate" the actual difference for optimal play. For analysis of actual games, this should probably be set to 1, while the tunes for match strength were consistently using 0.5 ... 0.6

In general, contempt 200 with attenuation 0.5 and contempt 100 with attenuation 1.0 will behave exactly the same, but since we expect `UCI_RatingAdv` to be set automatically, having this scaling factor is important.

Reply:
Maybe this? (generated by ChatGPT)

```
Scales Elo advantage used for contempt. It converts real Elo difference to simulated Elo
difference to optimize play. For analyzing actual games, use 1.0, and for match strength,
use 0.5-0.6.
```

What parameters do, should be more or less clear from the help description, and now it's not.
If it's not clear, people will come and ask, and then we'll have to explain it anyway. So it makes sense to write some description now. Even if it will be a bit lengthy (8-10 lines).

Reply:
Since the last update, attenuation doesn't scale the Elo difference itself anymore, but the effective "eval shift" caused by contempt. Thanks for the suggestet descriptions, I'll make it more specific and accurate.